src/Renci.SshNet/KeyboardInteractiveAuthenticationMethod.cs

```diff
@@ -125,9 +124,19 @@ private void Session_UserAuthenticationInformationRequestReceived(object sender,
 
                         var informationResponse = new InformationResponseMessage();
 
-                        foreach (var response in from r in eventArgs.Prompts orderby r.Id ascending select r.Response)
```

Comment:
Why are you removing ordering by `r.Id`? Is that intentional??

Reply:
The `Prompts` collection is read-only and the order never changes - it is redundant